.github/actions/private-tangle/setup/action.yml

```diff
@@ -3,7 +3,12 @@ description: 'Setup a private tangle'
 runs:
   using: "composite"
   steps:
-    - uses: mikefarah/yq@master
```

Comment:
This action is intended for running commands with `yq` but no inputs were supplied, so it wasn't doing anything. On the GitHub-hosted runners, `[yq` is already installed](https://github.com/actions/runner-images/blob/main/images/linux/scripts/installers/yq.sh) so we don't need to worry about installing it ourselves